### ISSUE TITLE
gray screen on boot & no ssh connection

### ISSUE THREAD

**Issue by Nick-bit-ui** [magdesign/PocketVJ-CP-exh#15]
**PocketVJ Exhibition Image and CP Version (if known)**

**Describe the bug**
My Pocket VJ exhibition isn't working. When I turn it on or plug it in, the loading screen briefly appears, but then it goes gray, and the test video mentioned in the instructions doesn't appear. I can connect to the Wi-Fi, but I can't find the page 2.0.0.100 (yes, it's http without 's', and yes, it's .100, as the Wi-Fi is also named .100). What could be the problem?

![image](https://github.com/user-attachments/assets/03218588-a7a3-4b0e-8e07-a5846eb1e0cc)

**Comment by magdesign** [comment]
sorry for the late replay.
since when does this problem happen, did you change something before it happend?
can you ssh into the unit?

edit:
i just remebered that this issue happend to people when beeing connected via lan to the internet and via wifi to the pocketvj. try disabling all other network connections and try again.

**Comment by Nick-bit-ui** [comment]
Hi, thank you for the response. The issue has been occurring since I bought it. I have never used it before. I tried SSH into the system, but it didn’t want to connect. I tried both via LAN and Wi-Fi, but the page just wouldn’t show up. I also tested it with my laptop without a LAN connection and with another laptop as well. Neither worked (also Tablet and Phone). However, I can connect to the Wi-Fi of the VJ Pocket with both devices. Is there something else I can try?

**Comment by magdesign** [comment]
hmmm... its several years since i used my last pocketvj 😅 (sorry for that).
maybe just reflash the image to the sd card (or to a new 32gb sdcard).
i know there was this edge case when ripping of the power while uploading a file, but i really cant remember how the fixing szenario was. did you check the [usermanual](https://pocketvj.com/manual/index.html).

here you get the [full image](https://github.com/magdesign/PocketVJ-CP-exh/releases/tag/latest).

see pdf on how to open:
[document.pdf](https://github.com/user-attachments/files/17246715/document.pdf)

**Comment by Nick-bit-ui** [comment]
Of course, I have read the entire user manual, and I’ve also reviewed all the additional information on GitHub and anywhere else I could find something. But no problem, and thank you very much for the support. Can I also flash it using Windows and the Pi Imager? I read in the manual that I need a Linux system. Is that still a requirement?

**Comment by magdesign** [comment]
not sure about windows..never used... there were people inserting the sd to a win computer and they ended up with a broken pocketvj image. i dont know what windows and/or the users did. maybe try it and report here. if it does not work, simply flash an ubuntu iso to an usb stick, boot into this stick (linux does run directly from a stick) and flash the image with gnome-disks (disks).
